src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModulePropertyDictionaryItemsController.cs

```diff
@@ -30,9 +34,13 @@ public class CatalogModulePropertyDictionaryItemsController : Controller
         /// <returns></returns>
         [HttpPost]
         [Route("search")]
-        [Authorize(ModuleConstants.Security.Permissions.Read)]
```

Comment:
Please leave empty [Authorize] attribute for this method. It is required to swagger ui.

Reply:
done